ideascube/templatetags/ideascube_tags.py

```diff
@@ -184,3 +184,11 @@ def remove_qs(context, **kwargs):
     params.update(MultiValueDict(existing))
     params.pop('page', None)  # Changing search context, reset page.
     return '?%s' % params.urlencode()
+
+
+@register.simple_tag
+def media(instance, attribute):
```

Comment:
The goal was to mirror the `static` tag. Eventually I would expect (and in fact hope) we don't know or care about what the tag does, we just get into the habit of always using for all media files, just like we use `static` for all static files.

This might not be a good idea of course, but this was my reasoning, and I still think it makes (some) sense. :smile: